src/edu/gatech/cs2340/trydent/TrydentEngine.java

```diff
@@ -0,0 +1,145 @@
+package edu.gatech.cs2340.trydent;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Main class that kicks off the TrydentEngine.
+ * 
+ * As a developer using the TrydentEngine, all you have to do is call
+ * TrydentEngine.start() to get started.
```

Comment:
Calling MyCoolLibrary.start() seems like an antiquated way of starting up libraries. But maybe that's just my opinion.

Is there a better way of doing this? Like a static initializer block or lazy initialization? Or is this considered good practice?